frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -150,11 +150,7 @@ export default function CourseDescription() {
               </Title>
               <Space direction="vertical" style={{ marginBottom: "1rem" }}>
                 <Text>
-                  <div
-                    dangerouslySetInnerHTML={{
-                      __html: course.raw_requirements,
```

Comment:
the reason we dangerously set html here is because the handbook API sends us raw HTML for this and we want to display the same way. We probably want to keep it how it was before.